web/premises/views.py

```diff
@@ -51,7 +51,10 @@ class ContentionDetailView(DetailView):
 
     def get_template_names(self):
         view = self.request.GET.get("view")
-        name = ("list_view" if view == "list" else "tree_view")
+        name = settings.DEFAULT_PREMISE_VIEW + "_view"
+        if view in ["list", "tree"]:
```

Comment:
I think,

it should be a constant. for example;

```python
# settings.py
DEFAULT_PREMISE_VIEW = 'tee'
PREMISE_VIEW_TYPES = {
    'list': 'premises/list_view.html',
    'tree': 'premises/tree_view.html'
}

# view.html
view_type = self.request.GET.get("view")
template_name = settings.PREMISE_VIEW_TYPES.get(view_type, settings.DEFAULT_PREMISE_VIEW)
return template_name
```

Reply:
Now it is more in a "convention over configuration" way of doing things. One setting less.

Reply:
@bahattincinic thank you for reviewing these BTW

Reply:
@bahattincinic would you settle for "convention over configuration" or should I incorporate your proposed change?